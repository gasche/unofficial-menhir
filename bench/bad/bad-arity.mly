%start<int> a
%token A
%%

a: b(A, A) {}

b(X): X {}
