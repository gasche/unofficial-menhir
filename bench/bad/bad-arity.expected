File "bad-arity.mly", line 5, characters 3-4:
Error: does this symbol expect 1 or 2 arguments?
